.markdownlintignore

```diff
@@ -0,0 +1,74 @@
+# Logs
```

Comment:
Does markdonwlint have default extensions to pick? If list the same files as in `.gitignore` and run `markdownlint .`, we might end up running markdownlint for anything including javsacript files, images etc.